src/kit/Select.tsx

```diff
@@ -121,7 +119,7 @@ const Select: React.FC<React.PropsWithChildren<SelectProps>> = forwardRef(functi
   const getPopupContainer = (triggerNode: Element) => {
     // triggerNode.parentElement can be falsy, so instead of a ternary, we fall back
     // to document.body
-    return (attachDropdownToContainer && triggerNode.parentElement) || document.body;
+    return triggerNode.parentElement || document.body;
```

Comment:
this needs to be optional -- if the parent element has hidden overflow, the popup will be cut off.